scripts/iam-tools/check_sso_http_status.py

```diff
@@ -65,8 +65,8 @@ def main(self):
             if sso_status != 200:
                 key_value += 1
 
-            zag.add_zabbix_keys({zabbix_key: key_value})
-            zag.send_metrics()
+                zag.add_zabbix_keys({zabbix_key: key_value})
+                zag.send_metrics()
 
```

Comment:
This looks wrong to me. Seems like these were already inside the loop, and now they'll only send when there's a failure. My previous change request was to move the key_value = 0 initialization inside the loop so that the key_value += 1 that could happen on the first loop pass won't affect the second check's value. Sorry if the GH review format didn't lend itself to knowing which line I was talking about.